### PR TITLE
Case 21010 - Tag domains and marketplace items as stand-alone device optimized

### PR DESCRIPTION
**Pull request by roxanneskelly**
https://highfidelity.manuscript.com/f/cases/21010/tag-domains-and-marketplace-items-as-quest-optimized

### CODE REVIEW

File: interface/resources/qml/hifi/commerce/purchases/PurchasedItem.qml [hunk 1]
```diff
@@ -417,7 +422,11 @@ Item {
                     verticalAlignment: Text.AlignVCenter;
 
                     onLinkActivated: {
-                        sendToPurchases({method: 'showPermissionsExplanation', itemType: root.itemType});
+                        if(link == "#standaloneIncompatible") {
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/commerce/purchases/PurchasedItem.qml]: coding standard: whitespace between if and paren

Comment (howard-stearns) [reply interface/resources/qml/hifi/commerce/purchases/PurchasedItem.qml]: also, ===, not ==.

File: interface/resources/qml/hifi/commerce/purchases/Purchases.qml [hunk 2]
```diff
@@ -110,6 +112,11 @@ Rectangle {
         }
     }
 
+    Component.onCompleted: {
+        isStandalone = PlatformInfo.isStandalone();
+        console.log(isStandalone ? "IS STANDALONE" : "ISN'T STANDALONE");
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/commerce/purchases/Purchases.qml]: ??

File: interface/src/Application.cpp [hunk 3]
```diff
@@ -3030,6 +3035,10 @@ void Application::initializeUi() {
     };
     OffscreenQmlSurface::addWhitelistContextHandler({
         QUrl{ "hifi/commerce/marketplace/Marketplace.qml" },
+        QUrl{ "hifi/commerce/purchases/Purchases.qml" },
```
Comment (howard-stearns) [inline interface/src/Application.cpp]: hmmm, how was this working before?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17999/HighFidelity-Beta-PR15142-c33d2aa.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17999/HighFidelity-Quest-Beta-PR15142-c33d2aa.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17999/HighFidelity-Beta-PR15142-c33d2aa.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17999/HighFidelity-Beta-Interface-PR15142-c33d2aa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17999/HighFidelity-Beta-PR15142-c33d2aa.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17999/HighFidelity-Beta-Interface-PR15142-c33d2aa.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18005/HighFidelity-Beta-PR15142-70cc659.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18005/HighFidelity-Quest-Beta-PR15142-70cc659.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18005/HighFidelity-Beta-PR15142-70cc659.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18005/HighFidelity-Beta-Interface-PR15142-70cc659.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18005/HighFidelity-Beta-PR15142-70cc659.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18005/HighFidelity-Beta-Interface-PR15142-70cc659.exe

**Pull request merged**